common/lib/xmodule/xmodule/partitions/tests/test_partitions.py

```diff
@@ -476,12 +476,12 @@ def test_get_user_group_id_for_partition(self):
 
         # get a group assigned to the user
         group1_id = self.partition_service.get_user_group_id_for_partition(self.user, user_partition_id)
-        self.assertEqual(group1_id, groups[0].id)
+        assert group1_id == groups[0].id
```

Comment:
^ should we feel empowered to use pytest asserts instead of unittest asserts from here on out? I understand why you did it, I am just curious about it as a policy (following up after our conversation w/ dave o, in which consistency was brought up)

Reply:
Yes, I think our current stance is "don't waste time trying to convert all the assertions, but feel free to use pytest assertions in new code or where the old assertions aren't providing enough useful information to diagnose known problems".